src/converse-notification.js

```diff
@@ -32,7 +32,8 @@ converse.plugins.add('converse-notification', {
             play_sounds: true,
             sounds_path: api.settings.get("assets_path")+'/sounds/',
             notification_icon: 'logo/conversejs-filled.svg',
-            notification_delay: 5000
+            notification_delay: 5000,
+            notify_nicknames_without_references: false
```

Comment:
When adding a new configuration option, please also document it here:
https://github.com/conversejs/converse.js/blob/master/docs/source/configuration.rst

Also, please update one of your tests to also check this configuration setting.